src/features/profile/DangerCard.js

```diff
@@ -64,6 +70,8 @@ const DangerCard = () => {
         alert(`Success, all your entries were deleted.`)
       }
     }
+    // call trackGoal here, when the user succeeds in deleting entries
+    trackGoal(FATHOM_DELETE_CONFIRMED);
```

Comment:
**suggestion:** This needs to be moved into the else above, so it is only triggered when it's successful.

Reply:
so it is only triggered when it's successful #303
e6a4025